tests/unit/Pirouette/Term/Syntax/SystemFSpec.hs

```diff
@@ -23,14 +23,14 @@ tests :: [TestTree]
 tests =
   [ testCase "Type-Type application, #1" $
       sameTy
-        [ty| \f : Type -> Type . (\x : Type . f x) (\w : Type . w) |]
-        [ty| \f : Type -> Type . f (\w : Type . w) |],
+        [ty| \f : * -> * . (\x : * . f x) (\w : * . w) |]
+        [ty| \f : * -> * . f (\w : * . w) |],
     testCase "Type-Type application, #2" $
       sameTy
-        [ty| \y : Type -> Type . \z : Type -> Type . (\x : Type . y x) (\w : Type . z w) |]
-        [ty| \y : Type -> Type . \z : Type -> Type . y (\w : Type . z w) |],
+        [ty| \y : * -> * . \z : * -> * . (\x : * . y x) (\w : * . z w) |]
```

Comment:
This one is hard to read with `*` instead of `Type`, for example.

Reply:
I think it is mostly due to lack of parenthesis and also we can omit the kind when it is `*`/`Type`. This can look like `[ty| \y : (* -> *) . \z : (* -> *) . (\x . y x) (\w . z w) |]`

Again, not a strong opinion for/against `*`/`Type`. I am ok to change back if this really bothers you.

Reply:
Rolled back to `Type`